device/tt_arch_types.h

```diff
@@ -12,11 +12,9 @@ namespace tt {
  * @brief ARCH Enums
  */
 enum class ARCH {
-    JAWBRIDGE = 0,
     GRAYSKULL = 1,
-    WORMHOLE = 2,
-    WORMHOLE_B0 = 3,
-    BLACKHOLE = 4,
+    WORMHOLE_B0 = 2,
```

Comment:
I wonder if this is encoded somewhere and this re-enumeration might break it. I doubt it though. No action here.

Reply:
It will uncover the bug where it is used in a wrong way :)